distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date.enso

```diff
@@ -692,7 +692,7 @@ type Date
          Format "2020-06-21" with French locale as "21. juin 2020"
 
              example_format = Date.new 2020 6 21 . format "d. MMMM yyyy" (Locale.new "fr")
-    @pattern make_date_format_selector
+    @pattern (value-> make_date_format_selector value)
```

Comment:
It seems that these two lines should be equivalent, right?

What is the reason for this change?

Reply:
Ah! It's because how defaults are handled, isn't it?

Reply:
Yep due to defaults for parse